listenbrainz/webserver/static/js/src/playlists/Playlists.tsx

```diff
@@ -159,20 +162,20 @@ export default class UserPlaylists extends React.Component<
         currentUser.auth_token,
         playlistId
       );
+      // Fetch the newly created playlist and add it to the state if it's the current user's page
```

Comment:
Comment looks outdated

Reply:
Mmm, the comment still describes what happens after it:
1. fetch the new playlist
2. on line 178 onwards : i on current user's page, add to state